app/views/games/show.html.erb

```diff
@@ -1,3 +1,41 @@
+<script type="text/javascript">
+$(document).ready(function () {
+  $("[data-piece-id]").children("[href]").children('img').draggable({
+    containment: ".chessboard",
+    create: function(){$(this).data('position',$(this).position())},
+    cursorAt:{left:15},
+    cursor:'move',
+    start: function( event, ui ) {
+      $(this).stop(true,true);
+    }
+  });
+
+  function snapToMiddle(dragger, target){
+      var topMove = target.position().top - dragger.data('position').top + (target.outerHeight(true) - dragger.outerHeight(true)) / 2;
+      var leftMove= target.position().left - dragger.data('position').left + (target.outerWidth(true) - dragger.outerWidth(true)) / 2;
+      dragger.animate({top:topMove,left:leftMove},{duration:600,easing:'easeOutBack'});
+  }
+  
+  $("td").droppable({
+      drop: function(event, ui) {
+        snapToMiddle(ui.draggable,$(this));
+        var x_pos = $(this).data('x-position');
+        var y_pos = $(this).parent().data('y-position');
+        var current_piece_id = ui.draggable.parent().parent().data('piece-id');
+        $.ajax({
+            url: "<%=current_game.id%>/move",
+            type: 'PUT',
+            dataType: 'json',
+            data: { piece_id: current_piece_id, x: x_pos, y: y_pos}
+        })
+        .done (function() {
+          location.reload(true); //refreshes page from server after move
```

Comment:
Just curious, but I wonder if the rendering of the board can be moved to a partial the way the rendering of _available_games is a partial attached to the welcome screen and have just the partial, IOW the board, reloaded. @jeffgerlach had you ever considered doing it that way? If nothing else something for a future refactor. Don't know if partials work that way.